funty 1.2.0 breaks tendermint-rs build

As per https://github.com/myrrlyn/funty/issues/3

**Steps to reproduce**

Build `master` with funty v1.2.0 and you should get such compiler errors:

```
error[E0034]: multiple applicable items in scope
   --> /home/thane/.cargo/registry/src/github.com-1ecc6299db9ec823/bitvec-0.18.4/src/field.rs:307:25
    |
307 |         check("load", len, M::BITS);
    |                               ^^^^ multiple `BITS` found
    |
note: candidate #1 is defined in the trait `BitMemory`
   --> /home/thane/.cargo/registry/src/github.com-1ecc6299db9ec823/bitvec-0.18.4/src/mem.rs:26:2
    |
26  |     const BITS: u8 = mem::size_of::<Self>() as u8 * 8;
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
note: candidate #2 is defined in the trait `IsNumber`
   --> /home/thane/.cargo/registry/src/github.com-1ecc6299db9ec823/funty-1.2.0/src/lib.rs:144:2
    |
144 |     const BITS: u32;
    |     ^^^^^^^^^^^^^^^^
help: disambiguate the associated constant for candidate #1
    |
307 |         check("load", len, BitMemory::BITS);
    |                            ^^^^^^^^^^^^^^^
help: disambiguate the associated constant for candidate #2
    |
307 |         check("load", len, IsNumber::BITS);
    |                            ^^^^^^^^^^^^^^
```

**What's the definition of "done" for this issue?**

When `master` builds again without breaking due to these errors.

--- a/tendermint/Cargo.toml
+++ b/tendermint/Cargo.toml
@@ -39,6 +39,8 @@ bytes = "1.0"
 chrono = { version = "0.4", features = ["serde"] }
 ed25519 = "1"
 ed25519-dalek = { version = "1", features = ["serde"] }
+# TODO(thane): Remove once https://github.com/myrrlyn/funty/issues/3 is fixed
+funty = "=1.1.0"
 futures = "0.3"
 num-traits = "0.2"
 once_cell = "1.3"

